custom_md_loader/classes/AppConstants.cls

```diff
@@ -1,3 +1,9 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license. 
+ * For full license text, see LICENSE.txt file in the repo root  or https://opensource.org/licenses/BSD-3-Clause
```

Comment:
I found a different OS license header in the current OS guidelines. I'll email it.